lib/cli/run-helpers.js

```diff
@@ -73,20 +75,59 @@ exports.list = str =>
   Array.isArray(str) ? exports.list(str.join(',')) : str.split(/ *, */);
 
 /**
- * `require()` the modules as required by `--require <require>`
+ * `require()` the modules as required by `--require <require>`.
+ *
+ * Returns array of `mochaHooks` exports, if any.
  * @param {string[]} requires - Modules to require
+ * @returns {MochaRootHookObject|MochaRootHookFunction} Any root hooks
  * @private
  */
-exports.handleRequires = (requires = []) => {
-  requires.forEach(mod => {
+exports.handleRequires = (requires = []) =>
+  requires.reduce((acc, mod) => {
     let modpath = mod;
     if (fs.existsSync(mod, {cwd}) || fs.existsSync(`${mod}.js`, {cwd})) {
```

Comment:
This call is wrong, the signature is: `fs.existsSync(path)`

Reply:
wow, huh.  okay.

Reply:
fixed this